app/controllers/admin/poems_controller.rb

```diff
@@ -1,5 +1,30 @@
 class Admin::PoemsController < ApplicationController
   def index
+    # instantiate poem for use in "Create poem" form (#create action)
+    @poem = Poem.new
+
     @poems = Poem.order(:id)
   end
+
+  def create
+    @poem = Poem.new(poem_params)
+
+    flash[:notice] =
+      if @poem.save
+        'Poem successfully created!'
+      else
+        'Poem creation failed.'
+      end
```

Comment:
Interesting, I had that structure at first but rubocop yelled at me about Conditional Assignment so I changed it to match the suggestion [here](http://www.rubydoc.info/github/bbatsov/RuboCop/RuboCop/Cop/Style/ConditionalAssignment).

Reply:
Weird, it looks like our Rubocop config is wrong. I'll have to change that in our LevelUp repos, too, since that's where I pulled our config from! Regardless, if you switch to using two different flashes ("notice" and "alert"), it should stop yelling at you with our existing config.